pkg/cloudprovider/aws/fake/ec2api.go

```diff
@@ -97,3 +98,93 @@ func (a *EC2API) DescribeSecurityGroupsWithContext(context.Context, *ec2.Describ
 	}
 	return &ec2.DescribeSecurityGroupsOutput{SecurityGroups: []*ec2.SecurityGroup{{GroupId: aws.String("test-group")}}}, nil
 }
+
+func (a *EC2API) DescribeInstanceTypesPagesWithContext(ctx context.Context, input *ec2.DescribeInstanceTypesInput, fn func(*ec2.DescribeInstanceTypesOutput, bool) bool, opts ...request.Option) error {
```

Comment:
if you make `fn` an errorable, you can return it directly
```
return fn(..., ...)
```

Reply:
by errorable, do you mean have `fn` be:

```
func(*ec2.DescribeInstanceTypesOutput, bool) err
```

instead of

```
func(*ec2.DescribeInstanceTypesOutput, bool) bool
```

If so, I can't do that since the ec2 api defines that func signature.

Reply:
Ahh gotcha. Nevermind.